public/v4/apps/pocketbase.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+
+services:
+  "$$cap_appname":
+    caproverExtra:
+      dockerfileLines:
+        - FROM alpine:3.16.2
+        - RUN apk add --no-cache unzip openssh
+        - ADD https://github.com/pocketbase/pocketbase/releases/download/v$$cap_version/pocketbase_$$cap_version_linux_amd64.zip /tmp/pb.zip
+        - RUN unzip /tmp/pb.zip -d /pb/
+        - EXPOSE 8080
+        - CMD ["/pb/pocketbase", "serve", "--http=0.0.0.0:8080"]
+      containerHttpPort: "8080"
+    volumes:
+      - "$$cap_appname-data:/pb/pb_data"
+
+caproverOneClickApp:
+  variables:
+    - id: "$$cap_version"
+      label: PrivateBin Version
```

Comment:
Should be renamed to `PocketBase Version`.

Reply:
Shame on me ... fix follow

Reply:
No worries, it happens to all of us :+1: